spec

```diff
@@ -3273,8 +3273,7 @@ endseg
 beginseg
     name "ovl_En_Vm"
     include "build/src/overlays/actors/ovl_En_Vm/z_en_vm.o"
-    include "build/data/overlays/actors/z_en_vm.data.o"
-    include "build/data/overlays/actors/z_en_vm.reloc.o"
```

Comment:
According to jenkins
```
+ ./tools/find_unused_asm.sh
data/overlays/actors/z_en_vm.data.s
data/overlays/actors/z_en_vm.reloc.s
```
you forgot to delete the asm source for those

Reply:
Thanks I fixed that and also made a few small changes.